nemo_text_processing/text_normalization/zh/data/char/upper_to_lower.tsv

```diff
@@ -0,0 +1,26 @@
+A	a
+B	b
+C	c
+D	d
+E	e
+F	f
+G	g
+H	h
+I	i
+J	j
+K	k
+L	l
+M	m
+N	n
+O	o
+P	p
+Q	q
+R	r
+S	s
+T	t
+U	u
+V	v
+W	w
+X	x
+Y	y
+Z	z
```

Comment:
Same question as the above. Is there any reason we need this mapping as a standalone file? There are lower() and upper() functions in Python.